beacon-chain/rpc/service.go

```diff
@@ -194,6 +194,125 @@ func NewService(ctx context.Context, cfg *Config) *Service {
 var _ stategen.CanonicalChecker = blockchain.ChainInfoFetcher(nil)
 var _ stategen.CurrentSlotter = blockchain.ChainInfoFetcher(nil)
 
+func (s *Service) initializeRewardServer(rewardsServer *rewards.Server) {
```

Comment:
It would be better to rename these functions to `initializeXXXRoutes` because that's what we essentially do here.